pkgs/applications/science/logic/hol/default.nix

```diff
@@ -81,8 +81,8 @@ stdenv.mkDerivation {
     '';
     homepage = "http://hol.sourceforge.net/";
     license = licenses.bsd3;
+    platforms = [ "x86_64-linux" ];
     maintainers = with maintainers; [ mudri ];
     platforms = with platforms; linux;
```

Comment:
Please remove one of the platforms which causes ofborg to call.

Reply:
I am an idiot ;D
didn't notice duplicated platform declaration)